### PR TITLE
"stats" target for make

### PR DESCRIPTION
**Pull request by phip1611**
This MR adds a convenient `make stats` command with an output like this:

image

### CODE REVIEW

**Comment by phip1611** [comment]
**ping** @l3nkz

File: Makefile [hunk 1]
```diff
@@ -53,9 +54,9 @@ watch: pdf
 # Cleans all intermediate files except for the produced pdf files.
 clean:
 	$(LATEXMK) -C
-	# latexmk does not clean up the aux-files produced by the chapters.
-	# They are stand-alone compilation units with a dedicated aux file.
-	# This comes because we include them with "\include" instead of "\input".
+	$(QUIET) latexmk does not clean up the aux-files produced by the chapters.
```
Comment (l3nkz) [inline Makefile:57]: `#` is missing here, if I am not mistaken?

File: Makefile [hunk 2]
```diff
@@ -72,3 +73,19 @@ check-french-spacing: $(DOC_TEX_ALL_SORTED)
...
+	$(QUIET)echo "    words:      $(shell detex $(DOC_TEX_ALL_SORTED) | sed '/^$$/d' | wc -w)"
+	$(QUIET)echo "    characters: $(shell detex $(DOC_TEX_ALL_SORTED) | sed '/^$$/d' | wc --chars)"
+	$(QUIET)echo
+	$(QUIET)echo "    Please note: There are possibly deviations compared to the final PDF as the compiled"
+	$(QUIET)echo "                 version might contain additional words, such as 'in Figure 3.5 on the"
+	$(QUIET)echo "                 next page' or the bibliography."
+	$(QUIET)echo
+	$(QUIET)echo "  \e[1mCompiled PDF:\e[0m"
+	$(QUIET)echo "    pages: $(shell pdfinfo $(DOC_PDF) | awk '/^Pages:/ {print $$2}') (total, not content pages)"
+	$(QUIET)echo "    words: $(shell pdftotext $(DOC_PDF) $(DOC_TEX:.tex=.txt) | cat $(DOC_TEX:.tex=.txt) | wc -w)"
```
Comment (l3nkz) [inline Makefile:89]: I think we should add the *.txt file that got created here to the clean target, so that it gets removed again.

**Comment by l3nkz** [comment]
I had a look at the code and suggested some changes. Do you want to fix them? Otherwise I will just fix them and merge next week.

File: Makefile (hunk 1 above)
Comment (phip1611) [reply Makefile:57]: good catch!

File: Makefile (hunk 2 above)
Comment (phip1611) [reply Makefile]: fixed

**Comment by phip1611** [comment]
please re-review :) @l3nkz

**Pull request merged**